tests/alternative-deps/WORKSPACE

```diff
@@ -162,3 +163,23 @@ http_archive(
 load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies")
 
 gazelle_dependencies()
+
+####################
+# Gazelle Cabal
+####################
+
+# QUESTION: When the pull request adding the 2 missing library functions in gazelle_cabal is merged,
+# should we refer to an explicit commit of gazelle_cabal or to `main`?
```

Comment:
Perhaps this question can be eliminated from all the files in which it appears and asked elsewhere where it will be answered.

I think it is fine to point to a commit hash and update it every now and then. Using `main` will cause the configuration to stop working when incompatibilities arise. If you'd like to detect this kind of breakage, I would put it in a separate job in CI, but it would be a task for another issue/pr.